src/matter/cluster/server/AttributeServer.ts

```diff
@@ -21,11 +22,16 @@ export class AttributeServer<T> {
     }
 
     set(value: T) {
-        if (value === this.value) return;
+        if (value === this.value) {
+            this.listeners.forEach(listener => listener(value, value));
```

Comment:
sneaky ... was thinking ... stop ... should one not be this.value .... but yes both are the same *gg*